src/query/ast/src/parser/expr.rs

```diff
@@ -1255,14 +1272,26 @@ pub fn expr_element(i: Input) -> IResult<WithSpan<ExprElement>> {
 
     let date_sub = map(
         rule! {
-            (DATE_SUB | DATESUB) ~ "(" ~ #interval_kind ~ "," ~ #subexpr(0) ~ "," ~ #subexpr(0) ~ ")"
+            DATE_SUB ~ "(" ~ #interval_kind ~ "," ~ #subexpr(0) ~ "," ~ #subexpr(0) ~ ")"
         },
         |(_, _, unit, _, interval, _, date, _)| ExprElement::DateSub {
             unit,
             interval,
             date,
         },
     );
+
+    let datesub = map(
+        rule! {
+            DATESUB ~ "(" ~ #interval_kind ~ "," ~ #subexpr(0) ~ "," ~ #subexpr(0) ~ ")"
```

Comment:
Since the argument parsing is basically the same, I chose different function names to distinguish between date sub. @sundy-li

Reply:
Rename to date_between